dod-advana/gamechanger-web
# Task/different urls

## Description
Navigating to different Gamechanger pages changes the URL allowing users to refresh the page or share a link to land on the same page.
## Tickets 
UOT-117008
## How to Test
-Navigate to the User Dashboard, Analyst Tools, Data Status Tracker, and About Us pages.
-Ensure the URL is changed for each page
-Refresh each page
